datafusion/sqllogictest/test_files/math.slt

```diff
@@ -596,20 +631,14 @@ select gcd(-9223372036854775808, 0);
 query error DataFusion error: Arrow error: Compute error: Signed integer overflow in GCD\(0, \-9223372036854775808\)
 select gcd(0, -9223372036854775808);
 
-# Test gcd with zero and negative values
```

Comment:
this was combined with other examples I think

Reply:
Yes. ` gcd(0, 0)` was duplicated with another test, and negative tests was moved too.